streams/src/main/java/org/apache/kafka/streams/Topology.java

```diff
@@ -506,7 +501,6 @@ public synchronized Topology addSink(final String name,
      * and write to its topic
      * @return itself
      * @throws TopologyException if parent processor is not added yet, or if this processor's name is equal to the parent's name,
```

Comment:
Oops! Good catch on these. We might as well drop the trailing commas on this and the following docs (like you did on the earlier ones).

Reply:
Fixed